fetch.bs

```diff
@@ -3652,8 +3652,20 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
     </ol>
 
    <li>
-    <p>Modify <var>httpRequest</var>'s
-    <a for=request>header list</a> per HTTP.
+    <p>If <var>httpRequest</var>'s <a for=request>header list</a> <a for="header list">contains</a>
+    `<code>Range</code>`, then <a for="header list">append</a>
+    `<code>Accept-Encoding</code>`/`<code>identity</code>` to <var>httpRequest</var>'s
+    <a for=request>header list</a>.
+
+    <p class="note no-backref"><a href="https://jakearchibald.github.io/accept-encoding-range-test/">Many
+    servers</a> mistakenly ignore `<code>Range</code>` headers if a non-identity encoding is
```

Comment:
No newline here. Only HTML allows newlines in inlines. I can fix this up for you before landing though.

Reply:
Ah, I thought the fetch rule was only for references. Fixed.